### PR TITLE
fix(foundryup): libusb path

### PR DESCRIPTION
**Pull request by sambacha**
```bash
$ otool -L /opt/homebrew/Cellar/libusb/1.0.26/lib/libusb-1.0.0.dylib
/opt/homebrew/Cellar/libusb/1.0.26/lib/libusb-1.0.0.dylib:
	/opt/homebrew/opt/libusb/lib/libusb-1.0.0.dylib (compatibility version 4.0.0, current version 4.0.0)
```

## Motivation
Fix libusb path detect
Foundryup should bail install after 3 network timeouts, this prevents some edge cases in CI environments and docker container environments

## Solution

wen v1.0

### CODE REVIEW

File: foundryup/install [hunk 1]
```diff
@@ -48,9 +64,11 @@ if [[ ":$PATH:" != *":${FOUNDRY_BIN_DIR}:"* ]]; then
 fi
 
 # Warn MacOS users that they may need to manually install libusb via Homebrew:
-if [[ "$OSTYPE" =~ ^darwin && ! -f /usr/local/opt/libusb/lib/libusb-1.0.0.dylib ]]; then
```
Comment (brockelmore) [inline foundryup/install:51]: We need to check both. M1s use the `/opt` path, intel macs use `/usr/local`

Comment (sambacha) [reply foundryup/install:51]: > We need to check both. M1s use the `/opt` path, intel macs use `/usr/local`

ACK

tho this also is using `.bashc`, not `.bash_profile` which is the correct source for user info if your using bash on macOS.

Comment (sambacha) [reply foundryup/install:51]: [`a94a0c2` (#3907)](https://github.com/foundry-rs/foundry/pull/3907/commits/a94a0c29ca01a9a3a49d1b6f7d0bea4a7ab6e3ba#diff-79a844de4d6869f243ecd1e7ca4d9532d9a4987f85ca63d9f4af85aa227ecb38R68-R101)

I refactored this - I think this approach more robust, LMK ser

Comment (sambacha) [reply foundryup/install:51]: > We need to check both. M1s use the `/opt` path, intel macs use `/usr/local`

@brockelmore think this is good

**Comment by gakonst** [comment]
Closing as stale.

**Comment by sambacha** [comment]
> Closing as stale.

How is this stale? Its ready for merge

**Pull request closed without merging**